src/conda.exe.spec

```diff
@@ -62,6 +62,13 @@ elif sys.platform == "darwin":
     ]
     extra_exe_kwargs["entitlements_file"] = os.path.join(HERE, "entitlements.plist")
 
+# Add .condarc file to bundle to configure channels
+# during the package building stage
+if "PYINSTALLER_CONDARC_DIR" in os.environ:
+    condarc = os.path.join(os.environ["RECIPE_DIR"], ".condarc")
+    if os.path.exists(condarc):
```

Comment:
Wouldn't it be more explicit to simply pass the path to the condarc and then copy it? Or is it to save the rename logic?

Reply:
As far as I know, pyinstaller does not support renaming, or at least not all versions do.